packages/react/src/dialog/root/useDialogRoot.ts

```diff
@@ -226,11 +233,18 @@ export interface SharedParameters {
    * @default true
    */
   dismissible?: boolean;
+  /**
+   * A ref to imperative actions.
+   */
+  action?: React.RefObject<{ unmount: () => void }>;
```

Comment:
The type of the ref should be exported so it's easy for consumers to pass a correct type to `React.useRef`

Reply:
Hmm, how should it be exported? Like `Dialog.ActionsRef`?

Reply:
Yeah, should be fine like that.